packages/core/components/FileDetails/FileDetails.module.css

```diff
@@ -173,19 +168,9 @@
 }
 
 .title {
-    font-size: large;
     margin: 0 0 0 var(--margin);
 }
 
 .icon-button span {
     font-weight: normal;
 }
-
-.icon-button:hover, .icon-button:active {
```

Comment:
Since these styles are removed, the hover states are misbehaving
<img width="407" alt="image" src="https://github.com/AllenInstitute/aics-fms-file-explorer-app/assets/24441706/102f703a-3a6b-4902-b4fb-4a71598c5c87">
<img width="452" alt="image" src="https://github.com/AllenInstitute/aics-fms-file-explorer-app/assets/24441706/b9e80490-b4ef-4525-aa25-9de7bcd22baf">

Reply:
RIP, will fix

Reply:
Fixed